opencxl/cxl/device/root_port_device.py

```diff
@@ -629,6 +639,12 @@ async def scan_component_registers(self, info: DeviceEnumerationInfo):
                         f"HDM Decoder Capability Offset: 0x{hdm_decoder_offset:08x}"
                     )
                 )
+            elif cxl_capability_id == 0x000B:
+                logger.info(self._create_message("Found BI Route Table Capability Header"))
+            elif cxl_capability_id == 0x000B:
+                logger.info(self._create_message("Found BI RT Capability Header"))
+            elif cxl_capability_id == 0x000C:
+                logger.info(self._create_message("Found BI Decoder Capability Header"))
```

Comment:
1. Please create an enum class.
2. line 642 and line 644 are checking for the same value.